apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.php

```diff
@@ -117,6 +117,7 @@ function enqueue_script() {
 		'A8C_ETK_ErrorReporting_Config',
 		array(
 			'shouldActivateSentry' => should_activate_sentry( get_current_user_id(), get_current_blog_id() ) ? 'true' : 'false',
+			'releaseName'          => 'WPCOM_' . WPCOM_DEPLOYED_GIT_HASH,
```

Comment:
I do still like the idea of being overly cautious and checking for the constant to be defined. I know we aren't loading this file on Atomic, but it wouldn't be hard for someone to load it and then cause fatal errors.

Just like `defined( 'WPCOM_DEPLOYED_GIT_HASH' ) ? 'WPCOM_' . WPCOM_DEPLOYED_GIT_HASH : null`

Reply:
> but it wouldn't be hard for someone to load it and then cause fatal errors.

The only way to load it would be to add a filter callback to `a8c_enable_error_reporting` that returns `true`. If someone does that for AT (i.e through `wpcomsh`) then the intention is clear - error reporting should be active, and the constant should also be defined. Both things should be done at the same time. I don't think the likelihood of accidental activation of error reporting is high. I think the code should indeed fail if the constant is not defined, as it's intrinsically needed for it to work. That said, I don't know if an undefined constant could cause a fatal error in WoA, so I don't have a strong opinion against adding the guard. I think that instead of null, we could instead return a string, perhaps the `wpcom-test-01` that's already created in Sentry.

Reply:
That makes sense to me. We could also use that in cases the hash has the other two fallback values (invalid or unknown). Maybe something really clear like WPCOM_NO_RELEASE? (Or similar!)

Reply:
I think `WPCOM_NO_RELEASE` makes sense. I'll add the ternary expression back and fallback to this string as you suggest 👍🏻

Reply:
> I think WPCOM_NO_RELEASE makes sense. I'll add the ternary expression back and fallback to this string as you suggest 

@noahtallen I added the ternary back. I also added the full `WPCOM_NO_RELEASE` string instead of trying to DRY by having the `WPCOM_` at the beginning of the expression to make it easier to find the actual string in the source-code if a future dev finds this release in Sentry and wants to understand where it came from.